.github/workflows/osx_testing.yml

```diff
@@ -0,0 +1,185 @@
+name: testing
+
+on: [push, pull_request]
+
+# TODO: Run testing using luarocks installed module.
+# TODO: Use caching of fixed tarantool versions or improve
+#       setup-tarantool to use it here.
+
+jobs:
+  testing_mac_os:
+    # We want to run on external PRs, but not on our own internal
+    # PRs as they'll be run by the push to the branch.
+    #
+    # The main trick is described here:
+    # https://github.com/Dart-Code/Dart-Code/pull/2375
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+
+    strategy:
+      fail-fast: false
+      matrix:
+        runs-on:
+          - macos-10.15
+          - macos-11.0
+        tarantool:
+          - brew
+          - 2.4.1
```

Comment:
There is no 1.10 versions by intention, due to #159. It worth to mention it explicitly here. Like 'add 1.10 versions after resolving gh-159'.

Reply:
Added this to the code. Shall I mention this TODO in commit message or PR description?

Reply:
Okay.